src/style.css

```diff
@@ -1,3 +1,77 @@
 body {
   margin: 0;
+  padding: 0;
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  box-sizing: border-box;
+
+  --main-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
+  padding: 0 50px;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.header_logo {
+  text-decoration: none;
+  display: flex;
+}
+
+.header_logo_img {
+  width: 40px;
+  height: 40px;
+  margin: 10px auto;
+}
+
+.nav {
+  display: flex;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__item {
+  display: flex;
+  margin: 0 0 0 20px;
```

Comment:
Be consistent with vertical margins. It's recommended to use only top or only bottom margins for elements to avoid unexpected margin collapsing. In this case, you might want to use `margin-right` instead of `margin-left` on `.nav__item`.